quantum/plugins/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ add_subdirectory(xasm)
 add_subdirectory(qpp)
 add_subdirectory(staq)
 
+file(GLOB HEADERS utils/*.hpp)
+target_include_directories(xacc PUBLIC utils)
```

Comment:
I don't think this is the right way to go. I think you should instead update all plugins target_include_directories to point to the utils directory.